### ISSUE TITLE
Stop spawning a spidertron remote

### ISSUE THREAD

**Issue by urda** [urda/quick-start-delight#7]
- See code [here](https://github.com/urda/quick-start-delight/blob/4cbbb445d0fd8c5ad2a1310436432c55bccfa64b/control.lua#L70-L72)
- Remotes are just "free" now in Factorio 2.0.
- FYI: The remotes do *not* appear to color-match their paired spidertron anymore.
- See also: #6

**Comment by urda** [comment]
Basically

```lua
inventory.insert({name = "spidertron-remote", count = 1})
```

can go.

**Comment by urda** [comment]
I think just setting the tech "spidertron" to true also makes the UI behave correctly with the new remote creation.

**Comment by urda** [comment]
This should be fixed via ca5fbb5ad881d90920b2ccb8601220b851cda219

**Comment by PiggyWhiskey** [comment]
Of note, this does not activate the use of Spidertron Remote which remains greyed out until the Spidertron research is complete.

Perhaps you could modify the shortcut "unavailable_until_unlocked" property
Factorio\data\base\prototypes\shortcuts.lua

```  
{
    type = "shortcut",
    name = "give-spidertron-remote",
    order = "e[spidertron-remote]",
    action = "spawn-item",
    localised_name = {"shortcut.make-spidertron-remote"},
    associated_control_input = "give-spidertron-remote",
    technology_to_unlock = "spidertron",
    unavailable_until_unlocked = true,
    item_to_spawn = "spidertron-remote",
    icon = "__base__/graphics/icons/shortcut-toolbar/mip/new-rts-tool-x56.png",
    icon_size = 56,
    small_icon = "__base__/graphics/icons/shortcut-toolbar/mip/new-rts-tool-x24.png",
    small_icon_size = 24
  },
```

**Comment by urda** [comment]
I'll take a look at that too, but I think based on my testing https://github.com/urda/quick-start-delight/commit/ca5fbb5ad881d90920b2ccb8601220b851cda219 commit fixed it. I'll be making a bug fix / 2.0 general release in the coming days or so when I'm not busy at work.
